Adopt apprepository-controller to the changes in kubeapps/chartsvc

murali-reddy opened the pull request:
New binary chart-repo supports both sync and delete commands. So use sync command in cronjob and jobs created for repo sync

prydonius commented on controller.go:
```diff
@@ -424,7 +424,7 @@ func jobSpec(apprepo *apprepov1alpha1.AppRepository) batchv1.JobSpec {
 					{
 						Name:    "sync",
 						Image:   repoSyncImage,
```
I think we probably want to also change the docker repo name to `kubeapps/chart-repo` or something. Now that @ngtuna's PR is merged I can push it there.

murali-reddy replied:
Sure. Please push the image and let me know, I can actually test end-to-end with proper image.

prydonius replied:
Sorry, got distracted. Just pushed kubeapps/chart-repo:latest :)

murali-reddy committed:
Adopt apprepository-controller to the changes in kubeapps/chartsvc

New binary chart-repo supports both sync and delete commands. So use
sync command in cronjob and jobs created for repo sync

murali-reddy committed:
On apprepositories.kubeapps.com CRD object delettion create a job
to clean up the charts for the repo from the DB

prydonius reviewed (approved):
lgtm, thanks!
